apps/ui/modules/search.ts

```diff
@@ -310,21 +317,7 @@ export async function getSearchOptions(
     value: n.pk ?? 0,
     label: getTranslationSafe(n, "name", lang),
   }));
-
-  const { data: unitData } = await apolloClient.query<
-    SearchFormParamsUnitQuery,
-    SearchFormParamsUnitQueryVariables
-  >({
-    query: SearchFormParamsUnitDocument,
-    variables: {
-      publishedReservationUnits: true,
-      ...(page === "direct" ? { onlyDirectBookable: true } : {}),
-      ...(page === "seasonal" ? { onlySeasonalBookable: true } : {}),
```

Comment:
are these filters not present in the new endpoint? they should be otherwise this introduces a regression.

Reply:
No, they don't seem to be present in `unitsAll`. I brought it up in #tilanvaraus-hyväksymistestaus - basically asking if it should/could be added to this query, too, to preserve the functionality which depends on them.

Reply:
Added this back, after changes to the backend